.github/workflows/test.yml

```diff
@@ -151,237 +154,50 @@ jobs:
-          --command="docker wait sync-to-checkpoint")
-
-          exit ${EXIT_CODE}
-
-      - name: Get state version from constants.rs
-        if: ${{ steps.sync-to-checkpoint.outcome == 'success' }}
-        run: |
-          STATE_VERSION=""
-
-          LOCAL_STATE_VERSION=$(grep -oE "DATABASE_FORMAT_VERSION: .* [0-9]+" $GITHUB_WORKSPACE/zebra-state/src/constants.rs | grep -oE "[0-9]+" | tail -n1)
-          echo "STATE_VERSION: $LOCAL_STATE_VERSION"
-
-          echo "STATE_VERSION=$LOCAL_STATE_VERSION" >> $GITHUB_ENV
-
-      - name: Get sync height from logs
-        if: ${{ steps.sync-to-checkpoint.outcome == 'success' }}
-        run: |
-          SYNC_HEIGHT=""
-
-          DOCKER_LOGS=$(\
-          gcloud compute ssh \
-          sync-to-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
-          --zone ${{ env.ZONE }} \
-          --quiet \
-          --ssh-flag="-o ServerAliveInterval=5" \
-          --command="docker logs sync-to-checkpoint --tail 20")
-
-          SYNC_HEIGHT=$(echo $DOCKER_LOGS | grep -oE 'flushing database to disk height=Height\([0-9]+\)' | grep -oE '[0-9]+' | tail -1 || [[ $? == 1 ]])
-
-          echo "SYNC_HEIGHT=$SYNC_HEIGHT" >> $GITHUB_ENV
-
-      # Create image from disk that will be used for the sync past mandatory checkpoint test
-      # Force the image creation as the disk is still attached even though is not being used by the container
-      - name: Create image from state disk
-        # Only run if the earlier step succeeds
-        if: ${{ steps.sync-to-checkpoint.outcome == 'success' }}
-        run: |
-          gcloud compute images create zebrad-cache-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}-v${{ env.STATE_VERSION }}-${{ env.NETWORK }}-checkpoint \
-          --force \
-          --source-disk=zebrad-cache-${{ env.GITHUB_SHA_SHORT }}-${{ env.NETWORK }}-checkpoint \
-          --source-disk-zone=${{ env.ZONE }} \
-          --storage-location=us \
-          --description="Created from commit ${{ env.GITHUB_SHA_SHORT }} with height ${{ env.SYNC_HEIGHT }}"
-
-      - name: Delete test instance
-        # If the `sync-to-checkpoint` step timeouts (+6 hours) the previous step (creating the image) willl be skipped.
-        # Even if the instance continues running, no image will be created, so it's better to delete it.
-        if: always()
-        continue-on-error: true
-        run: |
-          INSTANCE=$(gcloud compute instances list --filter=sync-to-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} --format='value(NAME)')
-          if [ -z "${INSTANCE}" ]; then
-            echo "No instance to delete"
-          else
-            gcloud compute instances delete "${INSTANCE}" --zone "${{ env.ZONE }}" --delete-disks all --quiet
-          fi
+    uses: ./.github/workflows/gcp-test-deploy.yml
+    with:
+      app_name: zebrad
+      test_id: sync-to-checkpoint
+      test_name: Test stateful disks rebuild
```

Comment:
I think skew between the test id which is the source of truth about what is being run and the test name could be confusing, maybe we skip including a test_name and instead of all humna-readable display names we can have `Run ${{ inputs.test_id }} test`

Reply:
I'm less opinionated about the `test_description` field

Reply:
Applied